src/Homie.cpp

```diff
@@ -19,6 +19,7 @@ HomieClass::HomieClass()
   Interface::get().reset.triggerTime = DEFAULT_RESET_TIME;
   Interface::get().reset.resetFlag = false;
   Interface::get().flaggedForSleep = false;
+  Interface::get().flaggedForSleep = false;
```

Comment:
Any particular reason to do this twice?

Reply:
It was an copy error. Fixed in the typo commit. Soz

Reply:
Just kidding, thx :)